### PR TITLE
Resolves #149 Contact doesn't do anything when clicked

### LINKED ISSUE #149: Contact doesn't do anything when clicked
On a new iphone, Contact doesn't do anything when clicked. Probably on a set up iPhone as well.

### PR DESCRIPTION
**Pull request by snuff4**
## PR Description

Added an alert when the iPhone doesn't have Mail App configured.

Please note that Mail service will not work in simulators. 

- [x] Fixes Issue #149 
  
 - [x] If you have multiple commits please combine them into one commit by [squashing](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) them. 
 - [ ] Code is well documented
 - [ ] Included unit tests for new functionality
 - [x] All user-visible strings are made translatable
 - [ ] Code passes Travis builds in your branch

### CODE REVIEW

**Comment by teolemon** [comment]
Tiny merge conflict in the settings @snuff4 @philippeauriach

**Comment by philippeauriach** [comment]
Resolved @teolemon

File: Sources/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -275,6 +275,9 @@
 "settings.sections.about" = "About";
 "settings.user-profile" = "Your contributor account";
 "settings.credits.title" = "Credits";
+"settings.contact.message" = "Thank you for contacting us.";
+"settings.contact-alert.title" = "Could not send E-mail";
+"settings.contact-alert.message" = "Your device could not send an E-mail.  Please check E-mail configuration and try again.";
```
Comment (teolemon) [inline Sources/en.lproj/Localizable.strings]:
```suggestion
"settings.contact-alert.message" = "Your device could not send an E-mail.  Please check your E-mail configuration and try again.";
```

**Comment by teolemon** [comment]
Ok. Thus Assuming it looks good to you @philippeauriach :-)

**Comment by snuff4** [comment]
img_0079

It was actually showing this before itself, I need to revert the PR and delete that function that I added

**Pull request merged**